compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIETileAndFuse.cpp

```diff
@@ -339,6 +341,10 @@ void AMDAIETileAndFusePass::runOnOperation() {
   SmallVector<int64_t> tileSizesVal;
   if (auto loweringConfig = getLoweringConfig(consumerOp)) {
     tileSizesVal = loweringConfig.getTileSizeVals(tilingLevel);
+  } else if (isa<linalg::GenericOp>(consumerOp)) {
```

Comment:
I think it might be better to just fix that upstream.

Reply:
I think this may be just logic I didnt implement here,
https://github.com/nod-ai/iree-amd-aie/blob/main/compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/KernelDispatch.cpp#L83
My rationale was to write minimal logic to get the matmul working. Handling the generic seems like a next step, @yzhang93 is your test case really hitting this else if part?

Reply:
oh makes sense since the pack decomposes the matmul as a generic.

Reply:
@MaheshRavishankar Yes, I'll try to fix that upstream. But before that is landed, we'll need to use this temporary hack solution.